basics/answers.md

```diff
@@ -296,6 +304,32 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(converted_numbers)
     ```
 
+5. ``` python
+   row_count = int(input("Type the number of rows needed:")) 
+   for column_count in range(1, row_count+1):
+       # Print numbers from 1 to column_count
+       for number in range(1, column_count+1): 
+           print(number, end=" ") 
+       print()  # creates a new line for the next row
+   ```
+   We actually use the first `for` loop to loop through the number of rows needed.Now,
+   through that `for` loop we got to which column the number should be inserted.
+   Now,the next `for` loop helps us to identify which numbers should be printed on the row which we currently accessed from the `for` loop above
+
+   
+  
+6. ```python
+   row_count=int(input("Type the number of rows needed:"))
+    
+   for column_count in range(1, row_count + 1): # first for loop
+     for number in range(column_count,row_count+1):# second for loop 
```

Comment:
I don't think `column_count` is a good variable name. If it is 1, we print 1,2,3,4,5, so 5 columns. Not 1 column, like you would expect.

I also changed the text below so it doesn't refer to "first for loop" etc, so the comments are no longer needed.

```suggestion
   for line_number in range(1, row_count + 1):
     for number in range(line_number, row_count+1):
```